install_config/topics/configuring_azure_using_ansible.adoc

```diff
@@ -0,0 +1,41 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='azure-configuring-ansible_{context}']
+= Configuring {product-title} for Azure using Ansible
+
+You can configure {product-title} for Azure at installation time or by running
+the xref:../install/configuring_inventory_file.adoc#configuring-ansible[Ansible
+inventory file] after installation.
+
+Add the following to the Ansible inventory file located at
+*_/etc/ansible/hosts_* by default to configure your {product-title} environment
+for Microsoft Azure:
+
+[source,yaml]
+----
+[OSEv3:vars]
+openshift_cloudprovider_kind=azure
+openshift_cloudprovider_azure_client_id=<app_ID> <1>
+openshift_cloudprovider_azure_client_secret=<secret> <2>
+openshift_cloudprovider_azure_tenant_id=<tenant_ID> <3>
+openshift_cloudprovider_azure_subscription_id=<subscription> <4>
+openshift_cloudprovider_azure_resource_group=<resource_group> <5>
+openshift_cloudprovider_azure_location=<location> <6>
+----
```

Comment:
This update is for 3.11? We have this pr merged: https://github.com/openshift/openshift-ansible/pull/9473, that means we will have more available parameters for ansible installation: openshift_cloudprovider_azure_cloud, openshift_cloudprovider_azure_vnet_name, and etc.

Reply:
This doc will be for 3.10. Depending on when the load balancer fix is shipped I will PR the new items in once the RPM is GA or close to GA